### ISSUE TITLE
Is the training process normal?

### ISSUE THREAD

**Issue by wenku1992abc** [philipperemy/deep-speaker#79]
![image](https://user-images.githubusercontent.com/33471245/113801707-f9ac5080-978b-11eb-8488-88afdf20398d.png)

is it overfitting?
the train dataset loss decrease quickly,but the val_loss even no change.
as shown in the picture 
@philipperemy

**Comment by wenku1992abc** [comment]
![image](https://user-images.githubusercontent.com/33471245/113815937-8c59e900-97a6-11eb-843c-51d90cab3cd7.png)
then,early stopping itself.

**Comment by philipperemy** [comment]
@wenku1992abc yes it looks like you overfitted a bit your data. Your accuracy is 99.18% and your validation accuracy is around 90%. That's why the early stopping kicked in. The model is very big so it requires a lot of data. However, I guess your training is completed. An accuracy of 90% or above can be acceptable if you train from scratch of many many speakers. If you have more data, use it!

**Comment by philipperemy** [comment]
@wenku1992abc yes it looks like you overfitted a bit your data. Your accuracy is 99.18% and your validation accuracy is around 90%. That's why the early stopping kicked in. The model is very big so it requires a lot of data. However, I guess your training is completed. An accuracy of 90% or above can be acceptable if you train from scratch of many many speakers. If you have more data, use it!

**Comment by philipperemy** [comment]
@wenku1992abc yes it looks like you overfitted a bit your data. Your accuracy is 99.18% and your validation accuracy is around 90%. That's why the early stopping kicked in. The model is very big so it requires a lot of data. However, I guess your training is completed. An accuracy of 90% or above can be acceptable if you train from scratch of many many speakers. If you have more data, use it!

**Comment by wenku1992abc** [comment]
thanks your reply！
by the way 1080 speaker is it less?
@philipperemy

**Comment by philipperemy** [comment]
LibriSpeech had 2484 speakers.

**Comment by wenku1992abc** [comment]
ok,thanks !
